src/libraries/Microsoft.PowerFx.Core/Lexer/TexlLexer.cs

```diff
@@ -969,11 +978,27 @@ internal static void ChoosePunctuators(ILanguageSettings loc, out string dec, ou
 
         private sealed class LexerImpl
         {
+            /*
```

Comment:
>/* [](http://example.com/codeflow?start=12&length=2)

nit: can we use //
We generally avoid multiline.

Reply:
Fixed